lib/parse-args.js

```diff
@@ -154,6 +174,10 @@ function buildYargs (withCommands = false) {
     })
     .pkgConf('c8')
     .demandCommand(1)
+    // Todo:  These next 6 lines might need to get removed
+    // This logic is covered with a default argument in
+    // the config options declaration in this file for yargs
+    /* c8 ignore next 6 */
     .check((argv) => {
```

Comment:
I don't understand the reason for introducing the new coerce logic, vs., just adding a test to cover the `check` behaviour.

I would rather we don't introduce coerce behaviour to the arguments.